.circleci/config.yml

```diff
@@ -8,41 +8,11 @@ executors:
     docker:
       - image: thecodingmachine/php:8.0-v4-cli
 jobs:
-  install-and-self-analyse:
+  "Code Style Analysis":
     executor: php-74
     steps:
       - checkout
-      - run: date "+%F" > /tmp/cachekey; cat composer.json >> /tmp/cachekey
-      - restore_cache:
-          keys:
-            - composer-v4
-      - restore_cache:
-          keys:
-            - psalm-cache-{{ checksum "/tmp/cachekey" }} # speeds up diff run
       - run: composer update
-
-      - save_cache:
-          key: composer-v4
-          paths:
-            - /home/docker/.composer/cache/files
-            - /home/docker/.composer/cache/vcs
-
-      - run:
-          name: Static analysis
-          command: php -dextension=pcntl.so ./psalm --threads=10
-
-      - save_cache:
-          key: psalm-cache-{{ checksum "/tmp/cachekey" }}
-          paths:
-            - /tmp/psalm
-
-      - persist_to_workspace:
-          root: /home/docker/project/
-          paths:
-            - .
-  "Code Style Analysis":
-    executor: php-74
-    steps:
       - attach_workspace:
           at: /home/docker/project/
```

Comment:
As it's the first task being executed is there any point of attaching a workspace?

Reply:
And there's no tasks left that persist anything to workspace.

Reply:
And subsequent tasks attaching a workspace would have it empty, I suppose.

Reply:
I removed it but to be totally honest, I don't know what I'm doing!

I thought this would be simpler to delete a single job :p

Any idea why the build fails?

Reply:
`persist_workspace` and `attach_workspace` is a way to share files between jobs. The build fails because it can't find the `bin/build-phar.sh` file. And the file is not there because the job neither did the checkout itself, nor attached a workspace that had this file.

Previously, `install-and-self-analyze` would do the checkout and then persist the checked out folder to the workspace that subsequent jobs were using. Now it doesn't happen.

Reply:
Thanks, I get it now